ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-master.yaml

```diff
@@ -49,6 +49,10 @@ tests:
   steps:
     cluster_profile: kubevirt
     workflow: openshift-e2e-kubevirt
+- as: e2e-kubevirt-ovn
+  steps:
+    cluster_profile: kubevirt
+    workflow: openshift-e2e-kubevirt-ovn
```

Comment:
As part of this change I would like to als rename e2e-kubevirt flow to e2e-kubevirt-sdn, so it will be clear from the job name

Reply:
Sure - This is only a validation 
Didn't opened that PR yet